### ISSUE TITLE
Medlemsystem

### ISSUE THREAD

**Issue by tellnes** [laafte/bull2#2]
- Medlemsstatus bør automatisk synces mot samfundets medlemsystem med noen untak
  - F.eks. Symforch har hatt medlemmer som ikke kvalifiserer til samfundet medlemskap (ikke fylt 18 enda)
  - Det vi gjør for å gi eksterne nøkkelkorttilgang til Låfte idag er å ta dem opp i en gjeng den dagen de skal ha tilgang og utløpsdato dagen etter. (Dette burde Sikring få ITK til å fikse en bedre ordning for)
  - Medlemmer som har permisjon burde ha grunnlegende tilgang selv om de ikke har fornyet medlemskapet sitt
  - Grupperingstilhørighet kan også synces fra MDB2

**Data tilknyttet et medlem:**
- [ ] Utmerkelser (g-nøkkel, polyhymnia og andre daljer)
- [ ] Liste over tidligere aktive semestre
- [ ] Liste over aktive verv på Låfte (det finnes mer enn styreverv)
- [ ] Kontaktinformasjon
  - E-postadresse
  - Postadresse
  - Telefonnummer
- [ ] Profilbilde
  - Skal være et hvor man kan gjennkjenne personen
  - Tidligere har dette vært fotografert og lagt inn av styret
- [ ] Det bør sendes ut en automatisk årlig e-post med spørsmål om oppdagering av kontaktinformasjon
- [ ] Automatisk synkronisering av medlemmer mot e-postlister
  - F.eks. egen liste for 2. fiolin

Tilgang til adminfunksjonalitet trenger ikke forenkles (f.eks. krav om ITK bruker), men tilgang til vanlig medlemsfunksjonalitet bør være superenkelt. Det er allerede mega komplisert for nye medlemmer med aktivering av medlemskort ol. Dette bør tenkes på som en helhetlig prosess. Dette bør kanskje skilles ut i et eget issue.

**Comment by ludvigsj** [comment]
> Medlemsstatus bør automatisk synces mot samfundets medlemsystem med noen untak

Dette er ikke teknisk mulig så vidt jeg har forstått ITK.

> Det vi gjør for å gi eksterne nøkkelkorttilgang til Låfte idag er å ta dem opp i en gjeng den dagen de skal ha tilgang og utløpsdato dagen etter. (Dette burde Sikring få ITK til å fikse en bedre ordning for)

Har vel strengt tatt ingenting med bull å gjøre?

**Comment by tellnes** [comment]
> > Medlemsstatus bør automatisk synces mot samfundets medlemsystem med noen untak
> 
> Dette er ikke teknisk mulig så vidt jeg har forstått ITK.

Det burde vel være mulig hvis vi får read access til postgres databasen med medlemslisten?

> > Det vi gjør for å gi eksterne nøkkelkorttilgang til Låfte idag er å ta dem opp i en gjeng den dagen de skal ha tilgang og utløpsdato dagen etter. (Dette burde Sikring få ITK til å fikse en bedre ordning for)
> 
> Har vel strengt tatt ingenting med bull å gjøre?

Hvis det synes så er det relevant.

**Comment by ludvigsj** [comment]
> > > Medlemsstatus bør automatisk synces mot samfundets medlemsystem med noen untak
> > 
> > Dette er ikke teknisk mulig så vidt jeg har forstått ITK.
> 
> Det burde vel være mulig hvis vi får read access til postgres databasen med medlemslisten?

Men det får vi altså ikke

**Comment by peterhgombos** [comment]
Det bør kanskje legges til en mulighet for å være pang i systemet, slik at oss som flytter fra byen fortsatt kan få informasjon hvis man ønsker det.

**Comment by tellnes** [comment]
@peterhgombos Hvilken informasjon mener du er relevant å formidle til ikke aktive panger. Det jeg egentlig spør om er, hvor skal grensen gå?

**Comment by peterhgombos** [comment]
Informasjon om konserter og lignende. Egentlig er det ut fra tanken om at det hadde vært fint med et nyhetsbrev for de som ikke er aktive lenger, men ønsker å kunne dukke opp når det skjer ting, enten konserter som blir holdt av Låfte, eller konserter på Låfte.

**Comment by tellnes** [comment]
Så et internbrev reservert kun for aktive og tidligere medlemmer da. Noen må jo lage dette brevet, så det er i så fall en styresak. Det burde kanskje komme et ønske fra noen som faktisk ikke er aktive om dette.

**Comment by ludvigsj** [comment]
Eller bare ha en generert rapport over kommende hendelser? Er ikke det mye greiere? Vi har jo allerede etablert at vi ønsker å beholde tidligere medlemmer i databasen, så er jo piece of cake å fikse tilgang til enkelte kommende arrangementer i kalenderen f.eks. for panger på ubestemt tid

**Comment by tellnes** [comment]
Jo, det er kanskje godt nok. Men på sikt er det trolig ønskelig å kunne gjøre tilpasninger av det som sendes ut.
